docs/content/intro/quickstart.md

```diff
@@ -200,37 +200,60 @@ brig login --insecure --server https://IP_ADDRESS --root
 ## Create a Project
 
 A Brigade [project] defines event handlers, such as the definition of a CI pipeline.
-In this example project, the handler prints a message using a string passed in the event payload.
 
-1. Download the example project to the current directory.
+1. Initialize a new Brigade project with the `brig init` CLI command.
 
-    **posix**
-    ```bash
-    curl -o project.yaml https://raw.githubusercontent.com/brigadecore/brigade/v2/examples/12-first-payload/project.yaml
     ```
-
-    **powershell**
-    ```bash
-    (New-Object Net.WebClient).DownloadFile("https://raw.githubusercontent.com/brigadecore/brigade/v2/examples/12-first-payload/project.yaml", "$pwd\project.yaml")
+    brig init --id first-project
+    ```
+    This will create a project definition like the following in `.brigade/project.yaml`, which defines a handler for the "exec" event and echoes "Hello, World!".
+
+    ```yaml
+    apiVersion: brigade.sh/v2-beta
+    kind: Project
+    metadata:
+    id: first-project
+    description: My new Brigade project
+    spec:
+    ## Subscribe below to any events that should trigger your script.
+    ## The commented example depicts a subscription to "exec"
+    ## events originating from the Brigade CLI. 
+    eventSubscriptions:
+    - source: brigade.sh/cli
+        types:
+        - exec
+    workerTemplate:
+        logLevel: DEBUG
+        defaultConfigFiles:
+        brigade.ts: |
+            import { events, Job } from "@brigadecore/brigadier"
+            
+            // Use events.on() to define how your script responds to different events. 
+            // The commented example below depicts handling of "exec" events originating 
+            // from the Brigade CLI.
+            
+            events.on("brigade.sh/cli", "exec", async event => {
+                let job = new Job("hello", "debian:latest", event)
+                job.primaryContainer.command = ["echo"]
+                job.primaryContainer.arguments = ["Hello, World!"]
+                await job.run()
+            })
+        
+        events.process()
     ```
-1. Open project.yaml
-
-    <script src="https://gist-it.appspot.com/https://raw.githubusercontent.com/brigadecore/brigade/v2/examples/12-first-payload/project.yaml"></script>
-
-    The project defines a handler for the "exec" event, that reads the event payload string and prints it out with "Hello, PAYLOAD!".
 
 1. Create the project in Brigade with the following command.
 
     ```
-    brig project create --file project.yaml
+    brig project create --file .brigade/project.yaml
     ```
 
 1. List the defined projects with `brig project list` and verify that you see your new project:
 
     ```console
     $ brig project list
```

Comment:
We probably need to rip through _all_ the docs and settle on a single convention-- `$` before commands or _no_ `$` before commands, but for now, let's just be consistent within this one page. None of the other commands have the leading `$` on this page, so let's remove it here also.